src/core/Decorations.ts

```diff
@@ -261,25 +258,24 @@ export default class Decorations {
   ) {
     const iconPath = join(this.extensionPath, "images", "hats", `${shape}.svg`);
     const rawSvg = readFileSync(iconPath, "utf8");
+    const { characterWidth, characterHeight } = fontMeasurements;
 
     const { originalViewBoxHeight, originalViewBoxWidth } =
       this.getViewBoxDimensions(rawSvg);
 
-    const hatWidthPx =
-      hatWidthToCharacterWidthRatio * fontMeasurements.characterWidth;
+    const hatWidthPx = hatWidthToCharacterWidthRatio * characterWidth;
     const hatHeightPx =
       (originalViewBoxHeight / originalViewBoxWidth) * hatWidthPx;
 
-    const svgWidthPx = Math.ceil(fontMeasurements.characterWidth);
-    const svgHeightPx =
-      fontMeasurements.characterHeight + hatHeightPx + hatVerticalOffset;
+    const svgWidthPx = Math.ceil(characterWidth);
+    const svgHeightPx = characterHeight + hatHeightPx + hatVerticalOffset;
 
     const newViewBoxWidth =
-      ((originalViewBoxWidth / hatWidthToCharacterWidthRatio) *
-        fontMeasurements.characterWidth) /
-      svgWidthPx;
```

Comment:
This was the dumb mistake.  We need to adjust from char width to svg width, not the other way around 😅